### PR TITLE
DOCS-2436: Change inclusion table header

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/build/configure/processes.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Navigate to that card.
 Then fill in the following fields:
 
 <!-- prettier-ignore -->
-| Attribute (Builder Mode) | Attribute (JSON) | Type    | Inclusion    | Description |
+| Attribute (Builder Mode) | Attribute (JSON) | Type    | Required? | Description |
 | ------------------------ | -------------------- | ------- | ------------ | ----------- |
```
Comment (npentrel) [inline docs/build/configure/processes.md:61]:
```suggestion
| Attribute (Builder Mode) | Attribute (JSON) | Type    | Required? | Description |
| ------------------------ | ----------------- | ------- | ---------- | ----------- |
```

File: docs/build/configure/processes.md [hunk 2]
```diff
@@ -57,8 +57,8 @@ Navigate to that card.
 Then fill in the following fields:
 
 <!-- prettier-ignore -->
-| Attribute (Builder Mode) | Attribute (JSON) | Type    | Inclusion    | Description |
-| ------------------------ | -------------------- | ------- | ------------ | ----------- |
+| Attribute (Builder Mode) | Attribute (JSON) | Type    | Required? | Description |
+| ------------------------ | ----------------- | ------- | ---------- | ----------- |
```
Comment (npentrel) [inline docs/build/configure/processes.md:61]:
```suggestion
| Attribute (Builder Mode) | Attribute (JSON) | Type    | Required? | Description |
| ------------------------ | ---------------- | ------- | ---------- | ----------- |
```

File: docs/build/micro-rdk/movement-sensor/gyro-mpu6050.md [hunk 3]
```diff
@@ -114,7 +114,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name                  | Type    | Inclusion    | Description |
+| Name                  | Type    | Required? | Description |
 | --------------------- | ------- | ------------ | ----------- |
```
Comment (npentrel) [inline docs/build/micro-rdk/movement-sensor/gyro-mpu6050.md:118]:
```suggestion
| --------------------- | ------- | --------- | ----------- |
```

File: docs/build/configure/processes.md [hunk 4]
```diff
@@ -57,8 +57,8 @@ Navigate to that card.
 Then fill in the following fields:
 
 <!-- prettier-ignore -->
-| Attribute (Builder Mode) | Attribute (JSON) | Type    | Inclusion    | Description |
-| ------------------------ | -------------------- | ------- | ------------ | ----------- |
+| Attribute (Builder Mode) | Attribute (JSON) | Type    | Required? | Description |
+| ------------------------ | ---------------- | ------- | ---------- | ----------- |
```
Comment (npentrel) [inline docs/build/configure/processes.md:61]:
```suggestion
| ------------------------ | ---------------- | ------- | --------- | ----------- |
```

File: docs/build/micro-rdk/movement-sensor/accel-adxl345.md [hunk 5]
```diff
@@ -117,7 +117,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name | Type   | Inclusion    | Description |
+| Name | Type   | Required? | Description |
 | ---- | ------ | ------------ | ----------- |
```
Comment (npentrel) [inline docs/build/micro-rdk/movement-sensor/accel-adxl345.md:121]:
```suggestion
| ---- | ------ | --------- | ----------- |
```

File: docs/components/arm/lite6.md [hunk 6]
```diff
@@ -91,7 +91,7 @@ Fill in the attributes as applicable to your arm, according to the table below.
 The following attributes are available for `lite6` arms:
 
 <!-- prettier-ignore -->
-| Attribute | Type | Inclusion | Description |
+| Attribute | Type | Required? | Description |
 | --------- | ---- | ----------| ----------- |
```
Comment (npentrel) [inline docs/components/arm/lite6.md:95]:
```suggestion
| --------- | ---- | --------- | ----------- |
```

File: docs/components/arm/ur5e.md [hunk 7]
```diff
@@ -73,7 +73,7 @@ Edit and fill in the attributes as applicable.
 The following attributes are available for `ur5e` arms:
 
 <!-- prettier-ignore -->
-| Attribute | Type | Inclusion | Description |
+| Attribute | Type | Required? | Description |
 | --------- | ---- | ----------| ----------- |
```
Comment (npentrel) [inline docs/components/arm/ur5e.md:77]:
```suggestion
| --------- | ---- | --------- | ----------- |
```

File: docs/components/arm/xarm6.md [hunk 8]
```diff
@@ -92,7 +92,7 @@ Edit and fill in the attributes as applicable.
 The following attributes are available for `xArm6` arms:
 
 <!-- prettier-ignore -->
-| Attribute | Type | Inclusion | Description |
+| Attribute | Type | Required? | Description |
 | --------- | ---- | ----------| ----------- |
```
Comment (npentrel) [inline docs/components/arm/xarm6.md:96]:
```suggestion
| --------- | ---- | --------- | ----------- |
```

File: docs/components/arm/xarm7.md [hunk 9]
```diff
@@ -92,7 +92,7 @@ Edit and fill in the attributes as applicable.
 The following attributes are available for `xArm7` arms:
 
 <!-- prettier-ignore -->
-| Attribute | Type | Inclusion | Description |
+| Attribute | Type | Required? | Description |
 | ----------| ---- | ----------| ----------- |
```
Comment (npentrel) [inline docs/components/arm/xarm7.md:96]:
```suggestion
| ----------| ---- | --------- | ----------- |
```

File: docs/components/component/model1.md [hunk 10]
```diff
@@ -123,7 +123,7 @@ Then remove and fill in the attributes as applicable to your COMPONENT, accordin
 The following attributes are available for `model1` arms:
 
 <!-- prettier-ignore -->
-| Name         | Inclusion    | Type   | Default Value | Description                                                                                                                                                                                      |
+| Name         | Required? | Type   | Default Value | Description                                                                                                                                                                                      |
 | ------------ | ------------ | ------ | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (npentrel) [inline docs/components/component/model1.md:127]:
```suggestion
| Name         | Required? | Type   | Default Value | Description |
| ---------- | ---------- | ------ | ------------- | ------------ |
```

File: docs/components/gripper/softrobotics.md [hunk 11]
```diff
@@ -90,7 +90,7 @@ Fill in the attributes as applicable to your gripper, according to the table bel
 The following attributes are available for `softrobotics` grippers:
 
 <!-- prettier-ignore -->
-| Name            | Inclusion    | Type   | Description                                                                                                                                                                                    |
+| Name            | Required? | Type   | Description                                                                                                                                                                                    |
 | --------------- | ------------ | ------ | -------- |
```
Comment (npentrel) [inline docs/components/gripper/softrobotics.md:94]:
```suggestion
| --------------- | --------- | ------ | -------- |
```

File: docs/services/data/trigger-sync.md [hunk 12]
```diff
@@ -106,7 +106,7 @@ To use [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at
 The following attributes are available for the `naomi:sync-at-time:timesyncsensor` sensor:
 
 <!-- prettier-ignore -->
-| Name    | Type   | Inclusion    | Description |
+| Name    | Type   | Required? | Description |
 | ------- | ------ | ------------ | ----------- |
```
Comment (npentrel) [inline docs/services/data/trigger-sync.md:110]:
```suggestion
| ------- | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/mpu6050.md [hunk 13]
```diff
@@ -72,7 +72,7 @@ Fill in the attributes as applicable to your movement sensor, according to the t
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name                  | Type    | Inclusion    | Description |
+| Name                  | Type    | Required? | Description |
 | --------------------- | ------- | ------------ | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/mpu6050.md:76]:
```suggestion
| --------------------- | ------- | --------- | ----------- |
```

File: docs/components/movement-sensor/merged.md [hunk 14]
```diff
@@ -99,7 +99,7 @@ Configure an array of the `name` of each movement sensor you want to add to your
   You can use the same sensor for multiple attributes if it supports multiple properties.
 
 <!-- prettier-ignore -->
-| Name                  | Type  | Inclusion                                | Description |
+| Name                  | Type  | Required? | Description |
 | --------------------- | ----- | ---------------------------------------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/merged.md:103]:
```suggestion
| --------------------- | ----- | ---------- | ----------- |
```

File: docs/components/gripper/softrobotics.md [hunk 15]
```diff
@@ -90,8 +90,8 @@ Fill in the attributes as applicable to your gripper, according to the table bel
 The following attributes are available for `softrobotics` grippers:
 
 <!-- prettier-ignore -->
-| Name            | Inclusion    | Type   | Description                                                                                                                                                                                    |
-| --------------- | ------------ | ------ | -------- |
+| Name            | Required? | Type   | Description                                                                                                                                                                                    |
+| --------------- | --------- | ------ | -------- |
```
Comment (npentrel) [inline docs/components/gripper/softrobotics.md:94]:
```suggestion
| --------------- | --------- | ------ | ------------ |
```

File: docs/components/movement-sensor/adxl345.md [hunk 16]
```diff
@@ -92,7 +92,7 @@ Fill in the attributes as applicable to your movement sensor, according to the t
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name | Type   | Inclusion    | Description |
+| Name | Type   | Required? | Description |
 | ---- | ------ | ------------ | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/adxl345.md:96]:
```suggestion
| ---- | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/adxl345.md [hunk 17]
```diff
@@ -105,7 +105,7 @@ Fill in the attributes as applicable to your movement sensor, according to the t
 Inside the `tap` object, you can include the following attributes:
 
 <!-- prettier-ignore -->
-| Name                | Type   | Inclusion    | Description |
+| Name                | Type   | Required? | Description |
 | ------------------- | ------ | ------------ | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/adxl345.md:109]:
```suggestion
| ------------------- | ------ | -------- | ----------- |
```

File: docs/components/movement-sensor/adxl345.md [hunk 18]
```diff
@@ -120,7 +120,7 @@ Inside the `tap` object, you can include the following attributes:
 Inside the `freefall` object, you can include the following attributes:
 
 <!-- prettier-ignore -->
-| Name                | Type   | Inclusion    | Description |
+| Name                | Type   | Required? | Description |
 | ------------------- | ------ | ------------ | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/adxl345.md:124]:
```suggestion
| ------------------- | ------ | -------- | ----------- |
```

File: docs/components/gripper/softrobotics.md [hunk 19]
```diff
@@ -90,8 +90,8 @@ Fill in the attributes as applicable to your gripper, according to the table bel
 The following attributes are available for `softrobotics` grippers:
 
 <!-- prettier-ignore -->
-| Name            | Inclusion    | Type   | Description                                                                                                                                                                                    |
-| --------------- | ------------ | ------ | -------- |
+| Name            | Required? | Type   | Description                                                                                                                                                                                    |
+| --------------- | --------- | ------ | ------------ |
```
Comment (npentrel) [inline docs/components/gripper/softrobotics.md:94]:
```suggestion
| --------------- | --------- | ------ | ----------- |
```

File: docs/components/gripper/softrobotics.md [hunk 20]
```diff
@@ -90,8 +90,8 @@ Fill in the attributes as applicable to your gripper, according to the table bel
 The following attributes are available for `softrobotics` grippers:
 
 <!-- prettier-ignore -->
-| Name            | Inclusion    | Type   | Description                                                                                                                                                                                    |
-| --------------- | ------------ | ------ | -------- |
+| Name            | Required? | Type   | Description                                                                                                                                                                                    |
```
Comment (npentrel) [inline docs/components/gripper/softrobotics.md:93]:
```suggestion
| Name            | Required? | Type   | Description |
```

File: docs/components/movement-sensor/adxl345.md [hunk 21]
```diff
@@ -105,8 +105,8 @@ Fill in the attributes as applicable to your movement sensor, according to the t
 Inside the `tap` object, you can include the following attributes:
 
 <!-- prettier-ignore -->
-| Name                | Type   | Inclusion    | Description |
-| ------------------- | ------ | ------------ | ----------- |
+| Name                | Type   | Required? | Description |
+| ------------------- | ------ | -------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/adxl345.md:109]:
```suggestion
| ------------------- | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/adxl345.md [hunk 22]
```diff
@@ -120,8 +120,8 @@ Inside the `tap` object, you can include the following attributes:
 Inside the `freefall` object, you can include the following attributes:
 
 <!-- prettier-ignore -->
-| Name                | Type   | Inclusion    | Description |
-| ------------------- | ------ | ------------ | ----------- |
+| Name                | Type   | Required? | Description |
+| ------------------- | ------ | -------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/adxl345.md:124]:
```suggestion
| ------------------- | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/cameramono.md [hunk 23]
```diff
@@ -97,7 +97,7 @@ Edit and fill in the attributes as applicable.
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name                       | Type   | Inclusion    | Description |
+| Name                       | Type   | Required? | Description |
 | -------------------------- | ------ | ------------ | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/cameramono.md:101]:
```suggestion
| -------------------------- | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/merged.md [hunk 24]
```diff
@@ -99,8 +99,8 @@ Configure an array of the `name` of each movement sensor you want to add to your
   You can use the same sensor for multiple attributes if it supports multiple properties.
 
 <!-- prettier-ignore -->
-| Name                  | Type  | Inclusion                                | Description |
-| --------------------- | ----- | ---------------------------------------- | ----------- |
+| Name                  | Type  | Required? | Description |
+| --------------------- | ----- | ---------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/merged.md:103]:
```suggestion
| --------------------- | ----- | --------- | ----------- |
```

File: docs/components/movement-sensor/imu-wit.md [hunk 25]
```diff
@@ -94,7 +94,7 @@ The `"serial_path"` filepath on a macOS system might resemble <file>"/dev/ttyUSB
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name  | Type   | Inclusion    | Description |
+| Name  | Type   | Required? | Description |
 | -------------- | ------ | ----------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/imu-wit.md:98]:
```suggestion
| ------ | ------ | ----------- | ----------- |
```

File: docs/components/movement-sensor/imu-wit-hwt905.md [hunk 26]
```diff
@@ -80,7 +80,7 @@ The `"serial_path"` filepath on a macOS system might resemble <file>"/dev/ttyUSB
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name  | Type   | Inclusion    | Description |
+| Name  | Type   | Required? | Description |
 | -------------- | ------ | ----------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/imu-wit-hwt905.md:84]:
```suggestion
| ----- | ------ | ----------- | ----------- |
```

File: docs/components/movement-sensor/gps-nmea.md [hunk 27]
```diff
@@ -154,7 +154,7 @@ For a movement sensor communicating over serial, you'll need to include a `seria
 For a movement sensor communicating over I<sup>2</sup>C, you'll need a `i2c_attributes` struct containing:
 
 <!-- prettier-ignore -->
-| Name            | Type   | Inclusion    | Description |
+| Name            | Type   | Required? | Description |
 | --------------- | ------ | ------------ | ------------------------ |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea.md:158]:
```suggestion
| --------------- | ------ | -------- | ------------------------ |
```

File: docs/components/movement-sensor/gps-nmea.md [hunk 28]
```diff
@@ -141,7 +141,7 @@ See examples of this struct in the example tabs above.
 For a movement sensor communicating over serial, you'll need to include a `serial_attributes` struct containing:
 
 <!-- prettier-ignore -->
-| Name               | Type   | Inclusion    | Description  |
+| Name               | Type   | Required? | Description  |
 | ------------------ | ------ | ------------ | ------------------------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea.md:145]:
```suggestion
| ------------------ | ------ | -------- | ------------------------- |
```

File: docs/components/movement-sensor/gps-nmea.md [hunk 29]
```diff
@@ -120,7 +120,7 @@ The `"serial_path"` filepath on a macOS system might resemble <file>"/dev/ttyUSB
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name              | Type    | Inclusion    | Description  |
+| Name              | Type    | Required? | Description  |
 | ----------------- | ------- | ------------ | ---------------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea.md:124]:
```suggestion
| ----------------- | ------- | -------- | ---------------- |
```

File: docs/components/movement-sensor/gps-nmea-rtk-pmtk.md [hunk 30]
```diff
@@ -109,7 +109,7 @@ Fill in the attributes as applicable to your movement sensor, according to the t
 The following attributes are available for a `gps-nmea-rtk-pmtk` movement sensor:
 
 <!-- prettier-ignore -->
-| Name                     | Type   | Inclusion    | Description |
+| Name                     | Type   | Required? | Description |
 | ------------------------ | ------ | ------------ | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea-rtk-pmtk.md:113]:
```suggestion
| ------------------------ | ------ | -------- | ----------- |
```

File: docs/components/movement-sensor/gps-nmea-rtk-pmtk.md [hunk 31]
```diff
@@ -109,8 +109,8 @@ Fill in the attributes as applicable to your movement sensor, according to the t
 The following attributes are available for a `gps-nmea-rtk-pmtk` movement sensor:
 
 <!-- prettier-ignore -->
-| Name                     | Type   | Inclusion    | Description |
-| ------------------------ | ------ | ------------ | ----------- |
+| Name                     | Type   | Required? | Description |
+| ------------------------ | ------ | -------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea-rtk-pmtk.md:113]:
```suggestion
| ------------------------ | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/gps-nmea-rtk-serial.md [hunk 32]
```diff
@@ -111,7 +111,7 @@ The `"serial_path"` filepath on a macOS system might resemble <file>"/dev/ttyUSB
 The following attributes are available for a `gps-nmea-rtk-pmtk` movement sensor:
 
 <!-- prettier-ignore -->
-| Name                     | Type   | Inclusion    | Description |
+| Name                     | Type   | Required? | Description |
 | ------------------------ | ------ | ------------ | ---------------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea-rtk-serial.md:115]:
```suggestion
| Name                     | Type   | Required? | Description |
| ------------------------ | ------ | -------- | ---------------- |
```

File: docs/components/movement-sensor/gps-nmea.md [hunk 33]
```diff
@@ -120,8 +120,8 @@ The `"serial_path"` filepath on a macOS system might resemble <file>"/dev/ttyUSB
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name              | Type    | Inclusion    | Description  |
-| ----------------- | ------- | ------------ | ---------------- |
+| Name              | Type    | Required? | Description  |
+| ----------------- | ------- | -------- | ---------------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea.md:124]:
```suggestion
| Name              | Type    | Required? | Description  |
| ----------------- | ------- | --------- | ------------ |
```

File: docs/components/movement-sensor/gps-nmea.md [hunk 34]
```diff
@@ -141,8 +141,8 @@ See examples of this struct in the example tabs above.
 For a movement sensor communicating over serial, you'll need to include a `serial_attributes` struct containing:
 
 <!-- prettier-ignore -->
-| Name               | Type   | Inclusion    | Description  |
-| ------------------ | ------ | ------------ | ------------------------- |
+| Name               | Type   | Required? | Description  |
+| ------------------ | ------ | -------- | ------------------------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea.md:145]:
```suggestion
| Name               | Type   | Required? | Description  |
| ------------------ | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/gps-nmea.md [hunk 35]
```diff
@@ -154,8 +154,8 @@ For a movement sensor communicating over serial, you'll need to include a `seria
 For a movement sensor communicating over I<sup>2</sup>C, you'll need a `i2c_attributes` struct containing:
 
 <!-- prettier-ignore -->
-| Name            | Type   | Inclusion    | Description |
-| --------------- | ------ | ------------ | ------------------------ |
+| Name            | Type   | Required? | Description |
+| --------------- | ------ | -------- | ------------------------ |
```
Comment (npentrel) [inline docs/components/movement-sensor/gps-nmea.md:158]:
```suggestion
| Name            | Type   | Required? | Description |
| --------------- | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/imu-wit-hwt905.md [hunk 36]
```diff
@@ -80,8 +80,8 @@ The `"serial_path"` filepath on a macOS system might resemble <file>"/dev/ttyUSB
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name  | Type   | Inclusion    | Description |
-| -------------- | ------ | ----------- | ----------- |
+| Name  | Type   | Required? | Description |
+| ----- | ------ | ----------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/imu-wit-hwt905.md:84]:
```suggestion
| Name  | Type   | Required? | Description |
| ----- | ------ | --------- | ----------- |
```

File: docs/components/movement-sensor/imu-wit.md [hunk 37]
```diff
@@ -94,8 +94,8 @@ The `"serial_path"` filepath on a macOS system might resemble <file>"/dev/ttyUSB
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name  | Type   | Inclusion    | Description |
-| -------------- | ------ | ----------- | ----------- |
+| Name  | Type   | Required? | Description |
+| ------ | ------ | ----------- | ----------- |
```
Comment (npentrel) [inline docs/components/movement-sensor/imu-wit.md:98]:
```suggestion
| Name  | Type   | Required? | Description |
| ----- | ------ | ---------- | ----------- |
```

File: docs/services/slam/orbslamv3/_index.md [hunk 38]
```diff
@@ -322,7 +322,7 @@ Setting `delete_processed_data: true` and `use_live_data: false` is invalid and
 Adjust these parameters to fine-tune the algorithm `orbslamv3` utilizes:
 
 <!-- prettier-ignore -->
-| Parameter Mode | Description | Inclusion | Default Value |
+| Parameter Mode | Description | Required? | Default Value |
```
Comment (JessamyT) [inline docs/services/slam/orbslamv3/_index.md:325]: Not sure but is this supposed to be this?
```suggestion
| Parameter | Description | Required? | Default Value |
```

File: static/include/services/cartographer/configparams.md [hunk 39]
```diff
@@ -1,5 +1,5 @@
 <!-- prettier-ignore -->
-| Parameter Mode | Description | Inclusion | Default Value | Notes |
+| Parameter Mode | Description | Required? | Default Value | Notes |
 | -------------- | ----------- | --------- | ------------- | ----- |
 | `mode` | `2d` | **Required** | None | |
```
Comment (JessamyT) [inline static/include/services/cartographer/configparams.md:4]:
```suggestion
| `mode` | `2d` | **Required** | None | | |
```
I'm surprised this isn't throwing an error but should it be like this?

**Review by JessamyT — APPROVED** [review]
LGTM, couple opt questions.

File: static/include/services/cartographer/configparams.md (hunk 39 above)
Comment (npentrel) [reply static/include/services/cartographer/configparams.md:4]: no - 5 columns

**Pull request merged**